src/Makefile.am

```diff
@@ -64,10 +65,15 @@ DISPATCH_CFLAGS=-Wall $(VISIBILITY_FLAGS) $(OMIT_LEAF_FP_FLAGS) \
 if DISPATCH_ENABLE_ASSERTS
 DISPATCH_CFLAGS+=-DDISPATCH_DEBUG=1
 endif
+
+if ANDROID
+ANDROID_LIBS= -L$(top_srcdir)/src/shims/android
```

Comment:
@frankeh is a committer on libpwq, so processing a pull request there should be fast.  We usually get good time turn around time from libkqueue (hours to days).  So, it's worth trying to do the right thing and maybe dropping this hunk from the PR